lib/install.go

```diff
@@ -114,7 +115,7 @@ func switchToVersion(product Product, tfversion string, binPath string, installP
 	}
 
 	logger.Infof("Switched %s to version %q", product.GetName(), tfversion)
-	addRecent(tfversion, installPath, product) //add to recent file for faster lookup
+	addRecent(tfversion, installPath, product) // add to recent file for faster lookup
```

Comment:
Not entirely sure why it's inline
```suggestion

	// add to recent file for faster lookup
	addRecent(tfversion, installPath, product)
```

Reply:
This is part of autoformatter and is not actually part of my changeset. Do you really want me to re-format it to multiline?

Reply:
Addressed this too: 73af0e2